src/classes/CONV_Account_Conversion_BATCH_TEST.cls

```diff
@@ -55,17 +55,17 @@ public with sharing class CONV_Account_Conversion_BATCH_TEST {
         CONV_Account_Conversion_BATCH_TEST.setPreHHAccountModel(true);
 
         // Insert 150 test Contacts with a different owner than the Current User
-        List<Contact> cons = UTIL_UnitTestData_TEST.CreateMultipleTestContacts(150);
+        List<Contact> cons = UTIL_UnitTestData_TEST.CreateMultipleTestContacts(50);
```

Comment:
@andrewyu-salesforce , let's use a var to store the number of contacts to create and then check in the following assert statements. The comment on line 57 still contains the "150" reference. Is there a reason to create 50 instead of 150 as before?

@force2b , is there a reason for creating 150 Contacts in the original test?

Reply:
According to GitBlame, it was set to 150 around [7 years ago](https://github.com/SalesforceFoundation/NPSP/commit/cd692c4dc290a9ae81698c0f12ecf5a9295313ee) for bulk testing, though I'm not really sure how 150 is truly bulk as compared to 50. I think of bulk testing more in the > 200 to allow for proper trigger recursion tests. That's tested by the TDTM_DMLgt200_TEST class though. So, for here I think 50 is as fine as 150, but I am curious why the change was made.

Reply:
It was updated to 50 because this tests consistently getting CPU time out during the development. I also think there is no difference between 150 and 50 because we already have the bulk test in place.

Reply:
@andrewyu-salesforce , changes with regards to "let's use a var to store the number of contacts to create and then check in the following assert statements. The comment on line 57 still contains the "150" reference" are still outstanding.